optimized_implementations/main.dart

```diff
@@ -0,0 +1 @@
+bool is_prime(int n){return false;}
```

Comment:
Using `_` for unused parameters is more idiomatic.
Also using the `=>` syntax would avoid a lot of overhead of `return`

Reply:
yup incorporated changes @RedRozio

Reply:
Ah I see that while I was checking the exact style guide someone was first. Although I so still sadly see one major flaw, false is one character slower than `0!=0` but I am not sure if it is worth opening another PR for such a small performance improvement